yaml/buildings.yaml

```diff
@@ -18,6 +18,12 @@ globals:
       roof_orientation: {col: "roof:orientation"}
       addr_housenumber: {col: "addr:housenumber"}
       addr_street: {col: "addr:street"}
+  - &building_kind_detail
+      kind_detail:
+        expr: mz_building_kind_detail(building)
+  - &building_part_kind_detail
+      kind_detail:
+        expr: mz_building_kind_detail("building:part")
```

Comment:
We already have `roof:shape` in tiles!

http://tangrams.github.io/explorer/#16.5/40.7104/-74.0123/roof_shape/dome